src/main/java/com/myaudiolibrary/web/model/Artist.java

```diff
@@ -0,0 +1,45 @@
+package com.myaudiolibrary.web.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+public class Artist {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "Name")
+    private String name;
+
+    @JsonIgnoreProperties("artist")
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "artist", orphanRemoval = true)
```

Comment:
Le `CascadeType.ALL` est effectivement ici utilisable et il faut bien ajouter le `orphanRemoval` afin que la suppression de l'album soit bien persistée

Plus d'infos [ici](https://openjpa.apache.org/builds/2.4.0/apache-openjpa/docs/jpa_overview_meta_field.html#jpa_overview_meta_cascade) et [là](https://stackoverflow.com/questions/306144/jpa-cascadetype-all-does-not-delete-orphans)